### PR TITLE
Mainmenu

### PR DESCRIPTION
**Pull request by danielab93**
#2 

I have created a main menu which is the user's ticket into my app. Please review the "MainMenu" component and also the "NavLinks" component

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -23,10 +23,7 @@ function App() {
         <NewFirstScreen />
```
Comment (anitaschiller) [inline src/App.jsx:23]: Cool that you use conditional rendering here 😎.

File: src/App.jsx [hunk 2]
```diff
@@ -23,10 +23,7 @@ function App() {
...
       ) : (
         <div>
-          <header>Test Test</header>
           <MainMenu />
-
-          <footer></footer>
         </div>
       )}
     </div>
```
Comment (anitaschiller) [inline src/App.jsx:29]: This comment is about line 40/37: Why do you use emotion here? I would recommend to stick to styled-components

File: src/components/MainMenu.jsx [hunk 3]
```diff
@@ -1,18 +1,87 @@
...
+import NavLinks from "./NavLinks";
+
+import Restaurants from "../pages/Restaurants";
+import Shopping from "../pages/Shopping";
+import Beauty from "../pages/Beauty";
+import Favourites from "../pages/Favourites";
+import Calculator from "../pages/Calculator";
+import NotFound from "../pages/NotFound";
 
 function MainMenu() {
```
Comment (anitaschiller) [inline src/components/MainMenu.jsx:13]: Is it really the main menu or rather the main page/index page?

File: src/components/MainMenu.jsx [hunk 4]
```diff
@@ -1,18 +1,87 @@
...
+          path="/"
+          element={
+            <>
+              <LogoImage />
+              <NavContainer>
+                <NavLinks />
+              </NavContainer>
+            </>
+          }
+        />
```
Comment (anitaschiller) [inline src/components/MainMenu.jsx:33]:
```suggestion
          element={<MainMenu />}
        />
```
Would that be the MainMenu? Consider extracting this part in one page/component.

File: src/components/MainMenu.jsx [hunk 5]
```diff
@@ -1,18 +1,87 @@
...
+const LogoImage = styled.img`
+  width: 20rem;
+  align-self: center;
+`;
+
+const NavContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 20%; // Elemente sollen sich gleichmäßig vertikal verteilen, mittig
```
Comment (anitaschiller) [inline src/components/MainMenu.jsx:65]: Is this comment necessary? If yes, please rewrite in English ;)

File: src/pages/NotFound.jsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+
+function NotFound() {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    setTimeout(() => {
+      navigate("/", { message: "Ups, hier geht es wohl nicht weiter" });
```
Comment (anitaschiller) [inline src/pages/NotFound.jsx:9]: Is this message being used on the destination page? If not, remove!

**Review by anitaschiller — CHANGES_REQUESTED** [review]
Hi Daniela,
nice work, structure looks nice! Just some remarks, see my comments. Just one more thing: please rename your GlobalStyles.jsx to GlobalStyles.js since there is no JSX in this file.

File: src/App.jsx (hunk 1 above)
Comment (danielab93) [reply src/App.jsx:23]: Thank you 😊

File: src/App.jsx (hunk 2 above)
Comment (danielab93) [reply src/App.jsx:29]: I forgot to delete it - sorry. That was content from the tutorial I've watched :)

File: src/components/MainMenu.jsx (hunk 3 above)
Comment (danielab93) [reply src/components/MainMenu.jsx:13]: I think it's both, the MainMenu and also the main page, I will rename it :)

File: src/components/MainMenu.jsx (hunk 4 above)
Comment (danielab93) [reply src/components/MainMenu.jsx:33]: I've extracted it, you can see it after the next push :)

File: src/components/MainMenu.jsx (hunk 5 above)
Comment (danielab93) [reply src/components/MainMenu.jsx:65]: I've deleted it, it's not necessary :)

File: src/pages/NotFound.jsx (hunk 6 above)
Comment (danielab93) [reply src/pages/NotFound.jsx:9]: Removed :)

**Comment by danielab93** [comment]
I've changed everything based on your comments and already merged the branch in main.
I think I'm not really sure if this was the right workflow but I will have more opportunities to practice the workflow :)

**Pull request merged**